tests/Stormpath/Tests/Cache/ResourceTest.php

```diff
@@ -0,0 +1,67 @@
+<?php
+/*
+ * Copyright 2013 Stormpath, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+namespace Stormpath\Tests\Resource;
+
+
+class ResourceTest extends \Stormpath\Tests\BaseTest {
```

Comment:
It would be good to see tests where we make calls several times (GET/POST/DELETE) and only the first time the server gets called and on the rest of the calls we prove that the results are obtained from the cache.